src/Symfony/Component/Messenger/Tests/RoutableMessageBusTest.php

```diff
@@ -41,18 +41,23 @@ public function testItRoutesToTheCorrectBus()
         $this->assertSame($envelope, $routableBus->dispatch($envelope, [$stamp]));
     }
 
-    public function testItExceptionOnMissingStamp()
+    public function testItRoutesToDefaultBus()
```

Comment:
Can you add a test to make sure it throws an exception if the default bus is not registered? 😉

Reply:
@sroze done